BE/toDoListAPI/src/main/java/school/mindera/toDoListAPI/model/DTOUpdatePosition.java

```diff
@@ -9,7 +9,8 @@
 @Setter
 @AllArgsConstructor
 @NoArgsConstructor
-public class DTOChangePosition {
+public class DTOUpdatePosition {
     private Integer taskId;
-    private Integer newPosition;
+    private Integer position;
+    private Integer parentId;
```

Comment:
Põe em maiscula por enquanto e dps muda se tudo, é q já tem em bues sitios assim no FE

Reply:
oq ta em maiscula?

Reply:
De qualquer maneira e melhor ficar assim, depois ainda mais trabalho vai dar, conforme for dando os erros trocasse

Reply:
o Parend Id mas yah ok,vou mudar no front end